packages/mdc-theme/_variables.scss

```diff
@@ -36,14 +36,20 @@ $mdc-theme-text-colors: (
     secondary: rgba(black, .54),
     hint: rgba(black, .38),
     disabled: rgba(black, .38),
-    icon: rgba(black, .38)
+    icon: rgba(black, .38),
+    textfield-background: rgba(black, .02),
```

Comment:
Adding component-specific values into mdc-theme maps seems really confusing; prior to these textfield changes, these maps had *only* purely generic theme values.

I suspect that we would rather parameterize these via a mixin within textfield itself, given the newer direction that color theming is taking in general?  (cc @acdvorak)

Reply:
For the sake of merge sanity, since there was a ton of flux in theming at the time, lets merge this and circle back when we have a complete implementation and methodology for theming.

Reply:
`mdc-theme-prop` can now accept color literals, so there shouldn't be a need to do this anymore?